recipes/links/build.sh

```diff
@@ -1,16 +1,6 @@
 #!/bin/sh
 set -eux -o pipefail
 
-./configure && make
-
-mkdir -p ${PREFIX}/bin/
-mkdir -p ${PREFIX}/bin/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/bin
-mkdir -p ${PREFIX}/bin/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/src
-
-cp src/LINKS_CPP ${PREFIX}/bin/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/src
-cp bin/* ${PREFIX}/bin/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/bin
-cp bin/LINKS* ${PREFIX}/bin/
-
-
-echo "#!/bin/bash" > ${PREFIX}/bin/LINKS-make
-echo "make -f $(command -v ${PREFIX}/bin/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/bin/LINKS-make) \$@" >> ${PREFIX}/bin/LINKS-make
```

Comment:
Yes looks like these two lines were dealing with the conda make the problem. So let's try

```
mv ${PREFIX}/bin/LINKS-make ${PREFIX}/bin/LINKS-make-real
echo "#!/bin/bash" > ${PREFIX}/bin/LINKS-make
echo "make -f $(command -v ${PREFIX}/bin/LINKS-make-real) \$@" >> ${PREFIX}/bin/LINKS-make
```